tsk/fs/logical_fs.cpp

```diff
@@ -608,7 +781,24 @@ add_directory_to_cache(LOGICALFS_INFO *logical_fs_info, const TSK_TCHAR *path, T
  * @return TSK_OK if successfull, TSK_ERR otherwise
  */
 static TSK_RETVAL_ENUM
-search_directory_recursive(LOGICALFS_INFO *logical_fs_info, const TSK_TCHAR * parent_path, TSK_INUM_T *last_inum_ptr, LOGICALFS_SEARCH_HELPER* search_helper) {
+search_directory_recursive(LOGICALFS_INFO* logical_fs_info, const TSK_TCHAR* parent_path, TSK_INUM_T* last_inum_ptr, LOGICALFS_SEARCH_HELPER* search_helper) {
+	return search_directory_recursive(logical_fs_info, parent_path, last_inum_ptr, NULL, LOGICAL_INVALID_INUM, search_helper);
```

Comment:
Same here. This is a really complex function. It would be great if we had a unit test for it.